Tests/Specs/crud/tests/write/updateMany-arrayFilters.json

```diff
@@ -97,7 +98,8 @@
       "outcome": {
         "result": {
           "matchedCount": 2,
-          "modifiedCount": 1
+          "modifiedCount": 1,
+          "upsertedCount": 0
```

Comment:
this line isn't actually here in the new file but I think @jmikola meant to add it along with adding it to all the other tests.
I resynced these because it was necessary to have the new field be present in order to decode to `UpdateResult`s in`CrudTests`

Reply:
I've since merged https://github.com/mongodb/specifications/pull/384 if you want to properly resync in a separate commit or PR.

When doing so, I find it's helpful to include the commit hash from the specs repo in the extended commit description.

Reply:
ok, I resynced all the CRUD files (minus those for the new count API) in a separate commit [here](https://github.com/mongodb/mongo-swift-driver/commit/6e3adbdf80ce6bc26d1783d041b1cd0cbbda4c02).